pkgs/development/python-modules/anyqt/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+}:
+
+buildPythonPackage rec {
+  pname = "anyqt";
+  version = "0.2.0";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "ales-erjavec";
+    repo = pname;
+    rev = "refs/tags/${version}";
+    hash = "sha256-dL2EUAMzWKq/oN3rXiEC6emDJddmg4KclT5ONKA0jfk=";
+  };
+
+  doCheck = false;
+
+  pythonImportsCheck = [ "AnyQt" ];
+
+  meta = with lib; {
+    description = "A PyQt/PySide compatibility layer";
+    homepage = "https://anyqt.readthedocs.io";
+    changelog = "https://github.com/ales-erjavec/anyqt/releases/tag/${version}";
+    downloadPage = changelog;
```

Comment:
To be able to do this you need to use the `rec` keyword before this attrset's opening brace.

Reply:
Thanks, but that didn't work. ofborg-eval is then running into issues since `maintainers` gets redefined.
I've just removed the `downloadPage` attribute if it was just a copy of `changelog` since it is not strictly necessary.